app/store/services/activities.js

```diff
@@ -234,18 +231,20 @@ export const computePerformance = (activities = {}) => {
         {
           name: "duration",
           percent: timeHeuristic,
-          value: overallPerformance.duration
+          value: overallPerformance.duration,
+          unit: "s"
         },
         {
           name: "pace",
           percent: paceHeuristic,
-          value: paceKilometerPerHour,
-          unit: "km/h"
+          value: paceMeterPerSecond,
+          unit: "m.s-1"
```

Comment:
m/s

Reply:
you should kidding me!!!!!!!
m/s === m.s-1